ci.sh

```diff
@@ -72,56 +86,32 @@ install_prerequisite() {
       pixman-static \
       pkgconf \
       python3 \
+      xz \
       zlib-static
   fi
 }
 
-clone_qemu_repository() {
-  git clone \
-    --branch v6.0.0 \
-    --depth 1 --recurse-submodules \
-    https://github.com/qemu/qemu
-}
-
-patch_qemu_for_alpine() {
-  [ $system = macos ] && return
-
-  pushd qemu > /dev/null
-
-  git apply << EOF
-diff --git a/include/hw/s390x/s390-pci-bus.h b/include/hw/s390x/s390-pci-bus.h
-index 49ae9f0..2bed491 100644
---- a/include/hw/s390x/s390-pci-bus.h
-+++ b/include/hw/s390x/s390-pci-bus.h
-@@ -82,7 +82,9 @@ OBJECT_DECLARE_SIMPLE_TYPE(S390PCIIOMMU, S390_PCI_IOMMU)
- #define ZPCI_EDMA_ADDR 0x1ffffffffffffffULL
-
- #define PAGE_SHIFT      12
--#define PAGE_SIZE       (1 << PAGE_SHIFT)
-+#ifndef PAGE_SIZE
-+    #define PAGE_SIZE       (1 << PAGE_SHIFT)
-+#endif
- #define PAGE_MASK       (~(PAGE_SIZE-1))
- #define PAGE_DEFAULT_ACC        0
- #define PAGE_DEFAULT_KEY        (PAGE_DEFAULT_ACC << 4)
-EOF
-
-  popd > /dev/null
+fetch_qemu() {
+  curl -O https://download.qemu.org/qemu-$QEMU_VER.tar.xz
+  xz -cd qemu-6.2.0.tar.xz | tar -xf -
```

Comment:
I think it's simpler to just do `tar xf qemu-6.2.0.tar.xz` and `tar` will automatically figure out which compression is used and do the decompression.

Reply:
alpine's didn't knew

Reply:
Ah, right, it's a simpler version of tar.